sdk/search/azure-search/pom.xml

```diff
@@ -28,13 +28,14 @@
   <dependencies>
     <dependency>
       <groupId>com.azure</groupId>
-      <artifactId>azure-core</artifactId>
-      <version>1.2.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <artifactId>azure-core-http-netty</artifactId>
+      <version>1.3.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-http-netty;current} -->
```

Comment:
Tags need to be updated to use `{x-version-update;unrleased_com.azure:azure-core-http-netty;dependency}`

Reply:
Shouldn't we use current instead of dependency?

Reply:
Made the changes.